app/assets/javascripts/account/AccountService.js.coffee

```diff
@@ -13,8 +13,11 @@ AccountService = ($state, $auth, $modal, $http, $translate, ShortFormApplication
   Service.myApplications = []
   Service.createdAccount = {}
   Service.rememberedShortFormState = null
-  Service.accountError = {message: null}
-  Service.accountSuccess = {message: null}
+  Service.accountError =
+    message: null
```

Comment:
YAY refactoring the alert-box and error/success messages! Notice that you have message and messages -- and some using message and messages. I kind of see the benefit of having both, but wondering if it's worth just using messages. I'm good either way.